src/index.d.ts

```diff
@@ -603,6 +603,15 @@ declare namespace RamdaAdjunct {
          */
         rejectP<T>(value?: T): Promise<T>;
 
+        /**
+         * Accepts a list of side-effecting functions and returns a new function. When
+         * invoked, this new function is applied to some arguments, each side-effecting
+         * function is applied to those same arguments in the order in which they were
+         * declared. The return of each side-effecting function is discarded and the
+         * new function will always return `undefined`.
+         */
+        sequencing(fns: Array<Function>): Function;
```

Comment:
let's primarily name this to `seq` (the function and the files) and add it alias `sequencing`.

```typescript
/**
 ... description
*/
seq(fns: Array<Function>): Function;
sequencing(fns: Array<Function>): Function; // alias
```

Reply:
Will do, was unsure if some future, more general purpose, function would take the name.